src/main/cpp/subsystems/SwerveDrive.cpp

```diff
@@ -152,18 +92,11 @@ void SwerveDrive::Periodic() {
     VOKC_CHECK(right_front_module_ != nullptr);
     VOKC_CHECK(right_back_module_ != nullptr);
 
-    VOKC_CALL(UpdateModules());
-   
-    // update odometry
-    VOKC_CHECK(swerve_odometry_ != nullptr);
-    swerve_odometry_->Update(frc::Rotation2d(units::degree_t(-heading)), *positions_); // negate heading because odometry expects counterclockwise to be positive, but the NavX is not
-
-    // update our position object
-    *position_ = swerve_odometry_->GetPose();
-
     if (in_auto) {
             // if we're just starting auto
         if (auto_state_ == INIT) {
+            std::cout << "INIT" << std::endl;
```

Comment:
Do we need this print?

Reply:
not really. it'd be useful for auto testing, but I can always throw it on a separate branch

Reply:
My preference is to not merge debug prints, and use more logging instead. Debug prints are good for tactical tests like you said